CONTRIBUTING.adoc

```diff
@@ -74,3 +74,75 @@ https://www.eclipse.org/projects/handbook/#resources-commit
 Contact the project developers via the project's "dev" list.
 
 * https://accounts.eclipse.org/mailing-list/data-dev
+
+=== Developer Certificate of Origin - DCO
+
+This is a security layer for the project and for the developers. It is mandatory.
+
+Follow one of these two methods to add DCO to your commits:
+
+==== 1. Command line
+
+Follow the steps:
+**Step 1:** Configure your local git environment adding the same name and e-mail configured at your GitHub account. It helps to sign commits manually during reviews and suggestions.
+
+[source, Bash]
+----
+git config --global user.name “Name”
+git config --global user.email “email@domain.com.br”
+----
+
+**Step 2:** Add the Signed-off-by line with the `'-s'` flag in the git commit command:
+
+[source, Bash]
+----
+$ git commit -s -m "This is my commit message"
+----
+
+==== 2. GitHub website
+
+You can also manually sign your commits during GitHub reviews and suggestions, follow the steps below:
+
+**Step 1:** When the commit changes box opens, manually type or paste your signature in the comment box, see the example:
+
+[source, Bash]
+----
+Signed-off-by: Name < e-mail address >
+----
+
+== Contact
+
+Contact the project developers via the project's "dev" list.
+
+* https://dev.eclipse.org/mailman/listinfo/data-dev
+
+
+== Issues
+
+If you found a bug or have an idea, check out the following sections before submitting your contribution.
+
+=== Check the issue tracker
+
+All our issues are centralized in our https://github.com/jakartaee/data[**main repository**], it is quite likely that you will find a topic that is being discussed. Check the https://github.com/jakartaee/data/issues[**open issues**], another good way to start is https://github.com/jakartaee/data/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22[**good first issues**].
```

Comment:
```suggestion
Because all of our issues are centralized in our https://github.com/jakartaee/data[**main repository**], it is quite likely that you will find a similar topic that is being discussed. Check the https://github.com/jakartaee/data/issues[**open issues**]. Another good way to start is https://github.com/jakartaee/data/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22[**good first issues**].
```